src/sass/login.scss

```diff
@@ -1,5 +1,190 @@
+    }
+  }
+
+  .mobile-explanation {
+    padding: 1em 0;
+
+    p {
+      margin-top: 0;
+    }
+  }
+
+  .container {
+    padding: 2em 0;
+  }
+
+  .logo {
+    text-align: center;
+    border-bottom: solid 2px $color-gray-light-alt;
+    padding: 1em 0;
+
+    a {
+      &:hover {
+        background-color: transparent;
+      }
+      display: inline-block;
+    }
+  }
+
+  .fed-warning {
+    font-size: 0.9em;
+    margin-bottom: 2em;
+
+    p {
+      margin: 0.5em 0;
+    }
+  }
+
+  .help-info {
+    padding: 2em 0;
+
+    p:first-of-type {
+      margin-top: 0;
+    }
+  }
+
+  .signin-actions-container {
+    text-align: center;
+    margin-right: 2em;
+
+    @include media-maxwidth($small-screen) {
+      margin-right: 0;
+    }
+
+    div:first-child {
+      background: $color-gray-light-alt;
+    }
+
+    border: solid 2px $color-gray-light-alt;
+
+    .top-banner {
+      text-align: center;
+      display: flex;
+      align-items: center;
+      padding: 0.25em 0;
+      font-style: italic;
+
+      div:first-child {
+        margin: 0 auto;
+      }
+
+      img {
+        height: inherit;
+        vertical-align: baseline;
+        padding: 0 0.25em;
+      }
+    }
+
+    .signin-actions {
+      padding: 2em 2em 1em;
+
+      @include media-maxwidth($small-screen) {
+        padding: 2em 1em 1em;
+      }
+
+      span.sidelines {
+        position: relative;
+        z-index: 1;
+        overflow: hidden;
+        text-align: center;
+        width: 100%;
+        display: inline-block;
+
+        font-size: 1.15em;
+        font-weight: bold;
+
+        &:before, &:after {
+          position: absolute;
+          top: 51%;
+          overflow: hidden;
+          width: 50%;
+          height: 2px;
+          content: "\a0";
+          background-color: $color-gray-light-alt;
+        }
+
+        &:before {
+          margin-left: -52%;
+          text-align: right;
+        }
+
+        &:after {
+          margin-left: 2%;
+        }
+      }
+
+      button {
+        display: flex;
+        justify-content: center;
+        align-items: flex-end;
+        width: 100%;
+        padding: 0.75em;
+        font-size: 16px;
+        margin: 1em 0;
+
+        &.dslogon {
+          background: $color-primary-darker;
+
+          &:hover {
+            background: $color-primary-darkest;
+          }
+        }
+
+        &.idme-create {
+          box-shadow: inset 0 0 0 2px $color-green;
+          color: $color-green;
+          margin-bottom: 0;
+
+          &:hover {
+            box-shadow: inset 0 0 0 2px $color-green-darker;
+            color: $color-green-darker;
+          }
+        }
+
+        img {
+          height: 1em;
+          padding: 0 0.25em;
+        }
+      }
+
+      .alternate-signin {
+        padding-top: 1em;
+
+        p:first-of-type {
+          margin-top: 0.5em;
+          color: $color-gray-dark;
+        }
+      }
+    }
+  }
+
+  .explanation-content {
+    padding: 0 2em;
```

Comment:
Padding on the buttons looks great. I think this line is still adding padding to the right-column content.

![image](https://user-images.githubusercontent.com/4106356/30353680-61e130ca-97dc-11e7-9dfd-63ef69f3ce7d.png)